lib/plugins/TaskRunner/WordPressApp.js

```diff
@@ -40,11 +40,11 @@ class WordPressApp extends LAMPApp {
 
     this.options.siteFolder = options.siteFolder || 'default';
     this.options.profileName = options.profileName || 'standard';
-    this.options.flushCaches = !!options.flushCaches || true;
+    this.options.flushCaches = String(options.flushCaches).toLowerCase() === 'true';
```

Comment:
I think so. I almost all cases you want the cache flushed when you set up a WP site on a different host. And we're doing all sorts of DB manipulation and URL rewriting so I can't imagine you'd wouldn't want the cache flushed. If you did, you'd probably have a very special reason for it.